src/main/java/graphql/language/AstPrinter.java

```diff
@@ -757,8 +781,11 @@ public static String printAst(Node node) {
      */
     public static void printAst(Writer writer, Node node) {
         String ast = printAst(node);
-        PrintWriter printer = new PrintWriter(writer);
-        printer.write(ast);
+        try {
```

Comment:
@kilink @bbakerman @dondonz why did we change that? What is the consequence of that?

Reply:
PrintWriter never throws an exception but just drops data.


This is better really. Change in behavior yes... but much better behavior

Reply:
Is that a breaking change needs to be documented?

Reply:
We can call out that it's a change in the release notes, I can add the label